src/main.cpp

```diff
@@ -2472,7 +2472,7 @@ int32_t ComputeBlockVersion(const CBlockIndex* pindexPrev, const Consensus::Para
 
     for (int i = 0; i < (int)Consensus::MAX_VERSION_BITS_DEPLOYMENTS; i++) {
         ThresholdState state = VersionBitsState(pindexPrev, params, (Consensus::DeploymentPos)i, versionbitscache);
-        if (state == THRESHOLD_LOCKED_IN || state == THRESHOLD_STARTED || state == THRESHOLD_ACTIVE) {
+        if (state == THRESHOLD_LOCKED_IN || state == THRESHOLD_STARTED || state == THRESHOLD_ACTIVE || !IsVersionBitRejected(params, (Consensus::DeploymentPos)i)) {
```

Comment:
should this be && instead of an || ? Otherwise just one of these things need to return true and the version bit will be concatenated. 
```
if ((state == THRESHOLD_LOCKED_IN || state == THRESHOLD_STARTED || state == THRESHOLD_ACTIVE) && !IsVersionBitRejected(params, (Consensus::DeploymentPos)i)) {
...
}
```
Or am i misunderstanding this?

Reply:
agree with @craigmacgregor